code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -398,4 +398,6 @@
 	if(!stored || stored.on_found(src))
 		return
 	stored.attack_hand(src) // take out thing from item in storage slot
+	if(delayed)
+		changeNext_move(CLICK_CD_MELEE)
```

Comment:
```suggestion
		changeNext_move(CLICK_CD_RANGE)
```
0.4 seconds instead of 0.8

still very reactable, but not significantly slower than just moving your mouse